SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+To understand which version will receive security patches we recommend you to read the [Grails Support Schedule](https://grails.org/support-schedule.html).
+
+## Supported Versions
+
+Below you can see the versions of Grails currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.2.x   | :white_check_mark: |
+| 5.1.x   | :x:                |
+| 5.0.x   | :x:                |
+| 4.1.x   | :white_check_mark: |
+| < 4.1   | :x:                |
+
+## Reporting a Vulnerability
+
+Please responsibly disclose (suspected) security vulnerabilities to
+**[The Grails Foundation](foundation@grails.org)**. You will receive a response from
```

Comment:
This doesn't seem to be rendering as expected with Github markdown view?

Should the link be `mailto:foundation@grails.org` ?

Reply:
Instead of putting the email as a link, I think we should just put the email address.  I also think we should use `security@grails.org` for this